virtcontainers/pkg/annotations/annotations.go

```diff
@@ -47,9 +47,6 @@ const (
 	// AssetHashType is the hash type used for assets verification
 	AssetHashType = vcAnnotationsPrefix + "AssetHashType"
 
-	// ConfigJSONKey is the annotation key to fetch the OCI configuration.
-	ConfigJSONKey = vcAnnotationsPrefix + "pkg.oci.config"
```

Comment:
Since this is an API change, we need to think about how we're going to manage its removal - if this PR is merged and we make a new release, some users may be surprised the feature has disappeared :smile:

Related: https://github.com/kata-containers/runtime/issues/1113#issuecomment-473825027

@egernst, @gnawux - any thoughts on this? Raise this issue in the AC meeting and announce the plan on the mailing list?

Reply:
@jodh-intel I don't think it's an API change. This annotation is used for storing OCI spec in kata-runtime itself's configuration. It's not exposed to user and user won't have any way to use it, so this is an internal change  and it's definitely safe to remove :smile:

Reply:
Sorry @WeiZhang555 ! You are of course correct - the lower-case `pkg.oci` prefix should have been the clue to me :smile: